src/Fantomas.Core/CodePrinter.fs

```diff
@@ -300,14 +300,19 @@ let genInheritConstructor (ic: InheritConstructor) =
         genType node.Type
         +> sepSpaceOrIndentAndNlnIfExpressionExceedsPageWidth (genExpr node.Expr)
 
-let isSynExprLambdaOrIfThenElse (e: Expr) =
+let isSynExprIfThenElse (e: Expr) =
```

Comment:
I think we want these functions in two flavours: partial active pattern and predicate function.
I would also not use `SynExpr` in the name here.